### PR TITLE
[Program: GCI] (Comment task - Part 1/2) prepare comment task database table. [Mentorship - Backend]

### LINKED ISSUE #425: comment a task
# Description:

**Is your feature request related to a problem? Please describe.**
As a User I can not comment task which makes completing task harder.

**Describe the solution you'd like**
As a User I would like to comment a task so I could ask/answer related to task questions.

**Describe alternatives you've considered**
Alternative is using chat.

**Additional context**
We need to refactor current tasks to be able to add comments to tasks.

# PRs:
[398](https://github.com/systers/mentorship-backend/pull/398)
[356](https://github.com/systers/mentorship-backend/pull/356)
[368](https://github.com/systers/mentorship-backend/pull/368)

### PR DESCRIPTION
**Pull request by asawesome07**
### Description
Added task comment model and DAO. This is part 1 of 2 of complete integration of task comment table.

Fixes #425

### Type of Change:
- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
I tested this using `python -m unittest discover tests`

Here's a screenshot
Capture

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [x] Update Postman API at /docs folder
- [x] Update Swagger documentation and the exported file at /docs folder

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Review by annabauza — APPROVED** [review]

**Comment by annabauza** [comment]
Can you also please resolve conflicts here? I'm happy to help if you are not sure.

**Comment by asawesome07** [comment]
> Can you also please resolve conflicts here? I'm happy to help if you are not sure.

yeah, could you help?

**Comment by asawesome07** [comment]
> > Can you also please resolve conflicts here? I'm happy to help if you are not sure.
> 
> yeah, could you help?

nevermind, made the changes, no conflicts now.

**Comment by annabauza** [comment]
Thanks, if you could please to check why test fails I would be grateful

**Comment by annabauza** [comment]
Thanks, if you could please to check why test fails I would be grateful

**Comment by asawesome07** [comment]
> Thanks, if you could please to check why test fails I would be grateful

it looks like, I merged the user_dashboard_details as well, which I guess doesn't exist and causes the error. I would be grateful if you could help out, I am studying for a test that this tomorrow

**Comment by annabauza** [comment]
sure thing you mind to give me write permission to your repo?

**Comment by asawesome07** [comment]
> sure thing you mind to give me write permission to your repo?

I have sent you an invite

File: app/api/models/mentorship_relation.py [hunk 1]
```diff
@@ -50,6 +52,18 @@ def add_models_to_namespace(api_namespace):
...
+    'comment': fields.String(required=True, description='Comment value for task comment')
+})
+
+list_task_comment_response_body = Model('List task comment response model', {
+    'id': fields.Integer(required=True, description='Task comment ID'),
+    'user_id': fields.Integer(required=True, description='ID of user who made the task comment'),
+    'task_id': fields.Integer(required=True, description='Task ID'),
+    'creation_date': fields.Float(required=True, description='Task comment creation date in UNIX timestamp format'),
+    'modification_date': fields.Float(required=True, description='Latest task comment modification date in UNIX timestamp format'),
+    'comment': fields.String(required=True, description='Comment value for task comment')
```
Comment (annabauza) [inline app/api/models/mentorship_relation.py:65]: missing brackets here

**Review by annabauza — CHANGES_REQUESTED** [review]
I have found the issue.

**Review by annabauza — APPROVED** [review]

**Comment by annabauza** [comment]
closing due to #356 merge

**Pull request closed without merging**